.ci/jobs.t/2-packaging-vagrant.yml

```diff
@@ -0,0 +1,37 @@
+- job:
+    name: elastic+elasticsearch+%BRANCH%+packaging-vagrant
+    display-name: "elastic / elasticsearch # %BRANCH% - packaging tests on vagrant (experimental)"
```

Comment:
I thought we had decided to ditch the nested virtualization builds? That is what this is, right?

Reply:
We did, but this is essentially the same job running on bare metal.
It's a replacement for the existing bare metal jobs ( there's an experimental job already running ) . It make better use of the bare metal hardware bu spreading out to more workers.